packages/sandcastle/src/Gallery.css

```diff
@@ -3,10 +3,14 @@
   flex-direction: column;
   height: 100%;
 
+  &.hidden {
+    display: none;
+  }
+
   .filters {
     display: flex;
     align-items: center;
-    gap: 0.3em;
+    gap: 8px;
```

Comment:
Does stratakit have standardized spacing which we should be referencing as variables?

Reply:
Yes, but they were only recently added to the library. I will be doing a pass on all the CSS values like these in a separate styling/stratakit update PR